src/shared/translation/translationEnglish.json

```
@@ -1,710 +1,717 @@
+  "pages.PrivacyPolicy.header.6.Text.1": "Research findings will be published in publications or conferences solely in anonymized form, and at no point will it allow any identification of you as an individual. No personal data (such as name or contact information) will be published.",
+  "pages.PrivacyPolicy.header.6.Text.2": "You have the right to exercise the following rights at any time:",
+  "pages.PrivacyPolicy.header.6.Text.3": "In any case, your participation in the data collection is voluntary. You have the right to withdraw at any time. Refusal or withdrawal will not result in any disadvantages for you.",
+  "pages.PrivacyPolicy.header.6.Text.4": "For compliance with data protection regulations in the case of this study, HS Kempten is responsible.",
+  "pages.PrivacyPolicy.LearningAnalytics": "Learning Analytics",
+  "pages.PrivacyPolicy.Name & Surname": "First and Last Name",
+  "pages.PrivacyPolicy.PrivacySupport.Name": "Data Protection Officer of HS Kempten",
+  "pages.PrivacyPolicy.PrivacySupport.Text": "For inquiries regarding data protection issues, please contact the data protection officer. The data protection officer can be reached at the following contact details:",
+  "pages.PrivacyPolicy.Support.Text": "If you have any questions about the study, please contact the principal investigator. In the case of this study, the contact person is:",
+  "pages.projectdescription": "Project Description",
+  "pages.projectinformation": "Project Information",
+  "pages.topic": "Topic",
+  "post": "Posted",
+  "previousText": "Previous",
+  "projectHASKi": "Project HASKI",
+  "projecthaski": "Project HASKI",
+  "resultPagesText": "Results",
+  "searchButtonText": "Search",
+  "services.AuthProvider.connectivityerror": "Error while checking login status. Check your connectivity",
+  "services.AuthProvider.cookienotfound": "Cookie not found. Please login again",
+  "services.AuthProvider.logout": "Logout successful",
+  "sort": "Sort by:",
+  "spellcheckText": "Did you mean:",
+  "success": "Success",
+  "tooltip.openGlobalSettings": "Open global settings",
+  "tooltip.openSettings": "Open settings",
+  "topic": "Topic",
+  "universityKempten": "University of Applied Sciences Kempten",
+  "update": "Updated",
+  "warning": "Warning",
+  "accept": "Accept",
+  "decline": "Decline",
+  "agree": "Agree",
+  "components.PrivacyModal.afterReading": "After reading please accept:",
+  "components.PrivacyModal.TermsofService": "Terms of service and Privacy Policy",
+  "components.PrivacyModal.accepted": "Thank you for your trust. Data for better learning experience is now being collected.",
+  "components.PrivacyModal.declined": "Data is not being collected. Should you change your mind look in the settings."
```

Der erste Teil ergibt für mich zeitlich keinen Sinn, da der Satz nur aussagt, dass im aktuellen Moment keine Daten gesammelt werden.

```suggestion
  "components.PrivacyModal.declined": "Data will not be collected. Should you change your mind, look in the settings."
```